src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -20,7 +20,7 @@ import org.joda.time.DateTime
 object ErrorAggregator {
 
   private val allowedDocTypes = List("main", "crash")
-  private val outputPrefix = "error_aggregates/v1"
+  private val outputPrefix = "error_aggregates/v2"
```

Comment:
This [bug comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1357277#c17) has some info re: the deploy procedure.
Also, Databricks suggest to [start a separate job is](https://databricks.com/blog/2017/05/18/taking-apache-sparks-structured-structured-streaming-to-production.html) in case of a schema change.
I [empirically checked](https://github.com/mozilla/telemetry-streaming/issues/19) that adding measures doesn't require any particular handling, but that's not the case for dimensions.